mule/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: mule
+    version: 1.2
+
+source:
+    fn: mule.tar.gz
+    url: https://github.com/SciTools/mule/releases/download/v1.2/mule.tar.gz
```

Comment:
@corinnebosley - This won't be OK since the Github repository does not contain the mule source code, it only has the source code archives available for download. I think this was correct the first time.